module/main.py

```diff
@@ -0,0 +1,58 @@
+import turtle, line, rectangle, polygon, circle, ellipse
+
+
+class Figures:
+    def __init__(self, x, t):
+        self.__x = x
+        if x == 1:
```

Comment:
That is not a good approach at all. This code violates the open-closed principle
https://stackify.com/solid-design-open-closed-principle/
If I ask you to add new figure you'll have to go through all your code and make changes in all files. That's not good at all

Reply:
> That is not a good approach at all. This code violates the open-closed principle https://stackify.com/solid-design-open-closed-principle/ If I ask you to add new figure you'll have to go through all your code and make changes in all files. That's not good at all

Modified module structure according Open-closed principle & added dependency injection approach.